src/main/scala/com/microsoft/hyperspace/actions/CreateActionBase.scala

```diff
@@ -39,17 +39,21 @@ private[actions] abstract class CreateActionBase(dataManager: IndexDataManager)
       .getOrElse(dataManager.getPath(0))
   }
 
+  protected def getNumBucketsConfig(spark: SparkSession): Int = {
```

Comment:
Can we name this as `def numBucketsForIndex` and `def indexLineageEnabled` as well?